examples/fit_models.py

```diff
@@ -91,6 +95,10 @@ def train_model(hparams):
     if hparams['model_class'].lower() == 'segmenter':
         from daart.models import Segmenter
         model = Segmenter(hparams)
+    elif hparams['model_class'].lower() == 'tmlp':
+        from daart.models import TemporalMLP
+        print('TMLP model')
+        model = TemporalMLP(hparams)
```

Comment:
can get rid of this elif block, temporal MLPs can be fit using the `Segmenter` class with `hparams['backbone'] = 'temporal-mlp'`